doc/openocd.texi

```diff
@@ -11476,6 +11476,11 @@ The second argument configures how OpenOCD should use the selected trigger featu
 With no parameters, prints current trigger features configuration.
 @end deffn
 
+@deffn {Command} {riscv set_external_trigger} value
+Associate the supplied external trigger with the SMP halt group for the harts. When the external trigger
```

Comment:
Thank you - i've dropped the word "SMP" from the documentation.

Reply:
If we decide to keep the status quo -- keep using halt groups for SMP groups -- then a simple Tcl command like this looks all right to me. 

Few suggestions:

- One halt group can have multiple external triggers (1:N relationship). So I'd call the command `add` rather than `set`.
- Since the command is only used in the configuration phase (COMMAND_CONFIG), it looks unnecessary to have a `remove` command to remove the trigger from the group.
- If the current relationship between halt groups and SMP stays in place, then the command name can contain `smp` -- as an additional hint for the user that it will apply to the whole SMP group.
- The command should contain `halt` or `halt_group` somewhere it its name to indicate it is for halt groups, not for resume groups.

For example:

```tcl
[target_name] riscv smp_add_ext_trigger halt_group <ext_trigger_number>
# halt_group would remain the only supported trigger type for the time being
```